tutorials/plot_visualize_evoked.py

```diff
@@ -99,13 +99,14 @@
 # more information.
 #
 # We can also combine the two kinds of plots to on using the ``plot_joint``
-# method of Evoked objects. Notice the ``topomap_args`` and ``ts_args``
-# parameters of :func:`mne.Evoked.plot_joint`.
-# You can pass key-value pairs as a python dictionary that gets passed as
-# parameters to the topomaps and time series of the joint plot.
-evoked_r_aud.plot_joint()
-
-# For slightly more elaborate plots, use these ``topomap_args`` and ``ts_args``
+# method of Evoked objects. Called as-is (``evoked.plot_joint()``), this
+# function should give a stylish and informative display of spatio-temporal
```

Comment:
+100

On Tue, Apr 26, 2016 at 5:56 PM jona-sassenhagen <notifications@github.com>
wrote:

> In tutorials/plot_visualize_evoked.py
> <https://github.com/mne-tools/mne-python/pull/3177#discussion_r61112520>:
>
> > @@ -99,13 +99,14 @@
> >  # more information.
> >  #
> >  # We can also combine the two kinds of plots to on using the ``plot_joint``
> > -# method of Evoked objects. Notice the ``topomap_args`` and ``ts_args``
> > -# parameters of :func:`mne.Evoked.plot_joint`.
> > -# You can pass key-value pairs as a python dictionary that gets passed as
> > -# parameters to the topomaps and time series of the joint plot.
> > -evoked_r_aud.plot_joint()
> > -
> > -# For slightly more elaborate plots, use these ``topomap_args`` and ``ts_args``
> > +# method of Evoked objects. Called as-is (``evoked.plot_joint()``), this
> > +# function should give a stylish and informative display of spatio-temporal
>
> Feature Request: Gifs should render in-line.
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly or view it on GitHub
> <https://github.com/mne-tools/mne-python/pull/3177/files/ecc09b6d311104375062c9764508d60a5ab628f3#r61112520>
>